lib/modal/user.dart

```diff
@@ -0,0 +1,4 @@
+class User {
+  String userId;
```

Comment:
variable can be final, because you probably don't change it anywhere in runtime, also constructor optional parameter could have @required annotation, if it is correct in your application